tests/sandbox/test_sandbox_application.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+from pytest_check import check
+import requests
+import json
+
+
+from data import Actor
+from typing import Callable
+from utils import HttpMethod
+
+
+@pytest.mark.sandbox
+class TestSandboxApplication:
+    @pytest.mark.parametrize("actor", Actor)
```

Comment:
Is this going to repeat the test for all business functions? Is that what you intended?

Reply:
It was for now yup, more to highlight that was could rather than it necessarily being needed. We do include some logic to allow certain business functions in the sandbox application however so I think trying this against each business function does give some benefit 🙂